packages/bpk-component-datepicker/src/BpkDatepicker-test.js

```diff
@@ -221,18 +251,23 @@ describe('BpkDatepicker', () => {
         minDate={new Date(2010, 1, 15)}
         maxDate={new Date(2010, 2, 15)}
         date={new Date(2010, 1, 15)}
+        onOpenChange={onOpenChangeHook}
```

Comment:
Sorry - another minor thing. Could we rename this `onOpenChangeMock`, otherwise someone might be confused into thinking it has something to do with React Hooks?

Reply:
Yeah, good point, sorry about that. Fixed now